### PR TITLE
Python script for task 3

### PR DESCRIPTION
**Pull request by kavi-12**
__21__ 

#21 

#### Short description of what this resolves:

Sending email through python script 

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

File: Submissions/kavi-12/#task-3/send_email.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+smtp_obj.login(email,password)
+# Receiver's address
+send_add = input('receiver email address: ')
+
+# Sending mail
+from_add = email
+to_add = send_add
+subject = input('Subject:')
+message = input('message:')
+msg = 'Subject:' + subject + '\n' + message
```
Comment (Lokesh2703) [inline Submissions/kavi-12/#task-3/send_email.py:21]: Check yourself whether you are getting content or not in the email by running the code.

**Comment by kavi-12** [comment]
to send email through this script you cannot use your own email password instead you have to enable 2-step verification from google accounts and then generate app password and use that password to send mails

**Comment by kavi-12** [comment]
IMG_20200120_130654__1579507713_47 9 101 237

**Comment by kavi-12** [comment]
IMG_20200120_130338__1579507887_47 9 101 237

**Comment by Lokesh2703** [comment]
Create separate PRs for each task otherwise u will not get points accordingly

**Pull request closed without merging**